dotCMS/src/main/java/com/dotcms/util/pagination/Paginator.java

```diff
@@ -29,14 +23,14 @@
      * @param direction If the order is Asc or Desc
      * @return
      */
-    public abstract Collection<T> getItems(User user, String filter, int limit, int offset,
-                                           String orderby, OrderDirection direction, Map<String, Object> extraParams);
+    public abstract PaginatedArrayList<T> getItems(User user, String filter, int limit, int offset,
+                                                   String orderby, OrderDirection direction, Map<String, Object> extraParams) throws PaginationException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotcms.util.pagination.PaginationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 153 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)